code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -106,10 +106,10 @@
 	name = "Fireball"
 	spell_type = /obj/effect/proc_holder/spell/aimed/fireball
 
+/*
 /datum/spellbook_entry/rod_form
 	name = "Rod Form"
-	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form
-	cost = 40
+	spell_type = /obj/effect/proc_holder/spell/targeted/rod_form */
```

Comment:
It instacrits. It does not instakill. If someone is fast enough they can and will snag and pull the critted person away to medbay. Wizards are all about mobility. Of course they're gonna go through Rwall. They have like. Three or four other spells and items that let them bypass secured areas. Maybe even five if we count wands and staffs as separate. Six if we are counting Staff of Chaos's Fireball/Door chance.

Currently rodform is the only reliable way for a wizard to down people (and leave corpses, which is essential for artifacer/necrostone builds) barring Summon Lesser Guns/Arcane Blast. Fireball has a tendency to explode the wiz, lightning bolt is useless, has a charge up time and needs to be tweaked, tesla blast I haven't tried. And we aren't covering Shapeshift cause we're discussing that elsewhere or E-Nath cause that doesn't leave corpses.